### ISSUE TITLE
YC

### ISSUE THREAD

**Issue by yuchiuchao** [NCAlgebra/NC#18]
Hello, I followed the instruction for web based installation within the Mathematica environment, but got an error
[Untitled-4.pdf](https://github.com/NCAlgebra/NC/files/9375717/Untitled-4.pdf)
.

**Comment by Jannertje** [comment]
I got the same error. Mathematica 13.1.0.0 on MacOS 12.6 (21G115) on an M1 Max MacBook Pro.

**Comment by mcdeoliveira** [comment]
Please check out the paclet installation for the experimental v6.0.0 branch. All installation issues should be gone. Let me know if the problem still persists.

**Comment by yuchiuchao** [comment]
Hello,

Thank you.  Looks like I still have problem (see attached).

Cheers,
YC

**Comment by mcdeoliveira** [comment]
I could not see your attachment.

**Comment by yuchiuchao** [comment]
Attached again.

**Comment by mcdeoliveira** [comment]
You might want to send it to my personal email. Looks like github is not letting it through.

**Comment by yuchiuchao** [comment]
Sure.  I’m not sure I have your personal email.

YC

**Comment by mcdeoliveira** [comment]
mauricio at ucsd dot edu

**Comment by yuchiuchao** [comment]
Here goes.

**Comment by yuchiuchao** [comment]
Please install using the paclet version from here:

https://github.com/NCAlgebra/NC/tree/v6.0.0

Maurício

**Comment by mcdeoliveira** [comment]
I am assuming the new paclet solved your problem. Reopen the issue if not.

**Comment by yuchiuchao** [comment]
Hello,

Sorry I was distracted from this.  I just installed it via the paclet link.  I ran the tests.  It seems that the last one, << NCPOLYTESTGB`, does require independent restart of the kernel, so I suppose some of its definitions may be preset by other tests, but that is not a problem.  All it takes is a restart as recommended.  There are other warning messages not atypical while running Mathematica related to hitting parameter limits etc., which are understandable.  It appears that this test so far is successful.  I will start using it for sure.  Thanks!

YC
